superset-frontend/src/explore/components/AdhocMetricEditPopover.jsx

```diff
@@ -276,8 +335,28 @@ export default class AdhocMetricEditPopover extends React.Component {
               </Select>
             </FormGroup>
           </Tabs.TabPane>
+          <Tabs.TabPane key={SAVED_TAB_KEY} tab="Saved">
+            <FormGroup>
+              <FormLabel>
+                <strong>Saved metric</strong>
```

Comment:
```suggestion
                <strong>t(Saved metric)</strong>
```

Reply:
Done. I also added translations to other untranslated strings that hung around